packages/dart/npt_flutter/lib/pages/profile_form_page.dart

```diff
@@ -1,18 +1,25 @@
 import 'package:flutter/material.dart';
 import 'package:flutter_gen/gen_l10n/app_localizations.dart';
 import 'package:npt_flutter/features/profile_form/profile_form.dart';
+import 'package:npt_flutter/features/profile/models/profile.dart';
 import 'package:npt_flutter/widgets/npt_app_bar.dart';
 
+class ProfileFormPageArguments {
+  final String uuid;
+  final Profile? copyFrom;
+  ProfileFormPageArguments(this.uuid, {this.copyFrom});
+}
```

Comment:
This page takes a class for arguments instead of just the uuid string now, I propogated those changes throughout the code base.